MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+bazel_dep(name = "gazelle", version = "0.40.0")
+bazel_dep(name = "rules_go", version = "0.50.1")
+
+go_sdk = use_extension("@rules_go//go:extensions.bzl", "go_sdk")
+go_sdk.download(version = "1.22.7")
+
+go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+
+# All *direct* dependencies are required to be listed explicitly
+use_repo(go_deps, "com_github_aws_aws_sdk_go", "com_github_aws_aws_sdk_go_v2", "com_github_aws_aws_sdk_go_v2_config", "com_github_aws_aws_sdk_go_v2_feature_ec2_imds", "com_github_aws_aws_sdk_go_v2_service_kms", "com_github_azure_azure_sdk_for_go_sdk_azidentity", "com_github_azure_azure_sdk_for_go_sdk_storage_azblob", "com_github_brunoscheufler_aws_ecs_metadata_go", "com_github_buildkite_bintest_v3", "com_github_buildkite_go_pipeline", "com_github_buildkite_interpolate", "com_github_buildkite_roko", "com_github_buildkite_shellwords", "com_github_creack_pty", "com_github_datadog_datadog_go_v5", "com_github_denisbrodbeck_machineid", "com_github_dustin_go_humanize", "com_github_dustinkirkland_golang_petname", "com_github_gliderlabs_ssh", "com_github_go_chi_chi_v5", "com_github_gofrs_flock", "com_github_google_go_cmp", "com_github_google_go_querystring", "com_github_gowebpki_jcs", "com_github_khan_genqlient", "com_github_lestrrat_go_jwx_v2", "com_github_mattn_go_zglob", "com_github_oleiade_reflections", "com_github_opentracing_opentracing_go", "com_github_pborman_uuid", "com_github_puzpuzpuz_xsync_v2", "com_github_qri_io_jsonschema", "com_github_stretchr_testify", "com_github_urfave_cli", "com_google_cloud_go_compute_metadata", "dev_drjosh_zzglob", "in_gopkg_datadog_dd_trace_go_v1", "in_gopkg_yaml_v3", "io_opentelemetry_go_contrib_propagators_aws", "io_opentelemetry_go_contrib_propagators_b3", "io_opentelemetry_go_contrib_propagators_jaeger", "io_opentelemetry_go_contrib_propagators_ot", "io_opentelemetry_go_otel", "io_opentelemetry_go_otel_exporters_otlp_otlptrace", "io_opentelemetry_go_otel_exporters_otlp_otlptrace_otlptracegrpc", "io_opentelemetry_go_otel_sdk", "io_opentelemetry_go_otel_trace", "org_golang_google_api", "org_golang_x_crypto", "org_golang_x_exp", "org_golang_x_net", "org_golang_x_oauth2", "org_golang_x_sys", "org_golang_x_term", "tools_gotest_v3")
```

Comment:
Was this generated? Seems like a thing that ought to be one per line.

Reply:
Yes, very much generated! This was all via `gazelle`, so happy I didn't have to assemble by hand.

Reply:
Are things like this something `buildozer` can fmt / normalise?
I noticed when I used it to add some deps to another project, it reindented everything.

Reply:
Ah, it didn't do much, I guess it's not opinionated about long lines:

```diff
pda@pda-m3 ~/bk/agent $ buildozer format //MODULE.bazel:all
fixed /Users/pda/bk/agent/MODULE.bazel

pda@pda-m3 ~/bk/agent $ git diff
diff --git i/MODULE.bazel w/MODULE.bazel
index 826e0241..484f3b05 100644
--- i/MODULE.bazel
+++ w/MODULE.bazel
@@ -6,7 +6,6 @@ go_sdk.download(version = "1.22.7")

 go_deps = use_extension("@gazelle//:extensions.bzl", "go_deps")
 go_deps.from_file(go_mod = "//:go.mod")
-
 go_deps.gazelle_override(
     directives = [
         "gazelle:go_generate_proto false",
```

Reply:
Oh and I guess [`buildifier`](https://github.com/bazelbuild/buildtools/tree/main/buildifier) is more specifically what I was after…